src/app/modules/form/admin-form/admin-form.controller.ts

```diff
@@ -218,3 +220,23 @@ export const handleCountFormSubmissions: RequestHandler<
   // Successfully retrieved count.
   return res.json(countResult.value)
 }
+
+/**
+ * Allow submission in preview without Spcp authentication by providing default values
+ * @param {Object} req - Express request object
+ * @param {Object} res - Express response object
+ * @param {Object} next - the next expressjs callback
+ */
+export const passThroughSpcp: RequestHandler = (req, res, next) => {
```

Comment:
this should be in admin-form.middlewares.ts hehehe but nvm i didn't catch it before